src/components/shared/filterDropdownButton/filterContainer.css

```diff
@@ -12,3 +24,14 @@
 .submitButton {
   margin: 20px 0;
 }
+
+.actionable {
+  color: var(--color-primary-standard);
+  cursor: pointer;
+  font-weight: var(--font-weight-bold);
+  text-align: right;
+  position: relative;
+  top: 10px;
+  font-size: 12px;
+  margin: 3px 0;
+}
```

Comment:
Did you consider using `<TertiaryButton size="xs">` for "More/Less filters"? It will need some custom css for positioning, but hopefully less than this.
https://github.com/LiskHQ/lisk-hub/blob/4cdc501a6630b1124afdc97d50439d8e2cac1222/src/components/toolbox/buttons/demo.js#L17

Reply:
I tried this but the `TertiaryButton's` padding and its text alignment make it also a bit difficult to adjust to the design requirements.

Reply:
Got it. Thanks for clarification 👍